admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -1547,7 +1547,11 @@ var priceCalculation = (function() {
   var taxElem = $('#form_step2_id_tax_rules_group');
   var reTaxElem = $('#step2_id_tax_rules_group_rendered');
   var displayPricePrecision = priceHTElem.attr('data-display-price-precision');
-  var ecoTaxRate = ecoTaxElem.attr('data-eco-tax-rate');
+  var ecoTaxRate = Number(ecoTaxElem.attr('data-eco-tax-rate'));
+  if (isNaN(ecoTaxRate)) {
+    ecoTaxRate = 0;
+  }
+  ecoTaxRate = ecoTaxRate / 100;
```

Comment:
```suggestion
  if (isNaN(ecoTaxRate)) {
    ecoTaxRate = 0;
  } else {
    ecoTaxRate = ecoTaxRate / 100;
  }
```

No need to divide if it's 0